cpp/src/parquet/column_writer.cc

```diff
@@ -2319,12 +2343,113 @@ Status TypedColumnWriterImpl<FLBAType>::WriteArrowDense(
   return Status::OK();
 }
 
+template <typename DType>
+void TypedColumnWriterImpl<DType>::UpdateBloomFilter(const T* values,
+                                                     int64_t num_values) {
+  if (bloom_filter_) {
+    // TODO(mwish): Would it allocate too much memory? Would an std::array<uint64_t, 64>
+    //  better here?
+    std::vector<uint64_t> hashes(num_values);
```

Comment:
Would I use an `std::array` to avoid allocating temporary stack memory?

Reply:
Yes, that's a good idea. You can probably make it 256 values.

Reply:
done